app/code/Magento/Quote/Model/Quote/Item/Repository.php

```diff
@@ -89,23 +90,18 @@ public function getList($cartId)
     /**
      * @inheritdoc
      */
-    public function save(\Magento\Quote\Api\Data\CartItemInterface $cartItem)
```

Comment:
As mentioned on previous comments, let's not change the Repository implementation and do it in separate classes.